compliance/sparql/src/test/java/org/eclipse/rdf4j/sail/memory/MemorySPARQL10W3CQueryTest.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.sail.memory;
+
+import junit.framework.Test;
+import org.eclipse.rdf4j.query.Dataset;
+import org.eclipse.rdf4j.query.parser.sparql.manifest.SPARQL11ManifestTest;
+import org.eclipse.rdf4j.query.parser.sparql.manifest.SPARQLQueryTest;
+import org.eclipse.rdf4j.repository.Repository;
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+
+import java.net.URL;
+
+public class MemorySPARQL10W3CQueryTest extends SPARQLQueryTest {
```

Comment:
This duplicates `org.eclipse.rdf4j.query.parser.sparql.manifest.W3CApprovedSPARQL10QueryTest`.